exercises/movr/vehicles/src/main/resources/application.yaml

```diff
@@ -11,7 +11,7 @@ spring:
       value-deserializer: org.springframework.kafka.support.serializer.JsonDeserializer
       properties.spring.json.value.default.type: io.roach.movrapi.events.KafkaMessage
   datasource:
-    url: jdbc:postgresql://localhost:26257/movr_vehicles
+    url: jdbc:postgresql://localhost:26263/movr_vehicles
```

Comment:
I'm on your branch and still finding references to 26257. My shell prompt is everything before the `%` and includes a virtualenv (at the start) and the branch name (at the end before the `%`). Looks like inconsistent port usage at first glance but I thought I'd ask before I spend time checking: Does this look correct to you? If you want to add another commit before I continue, cool, and if it looks right to you, then I'll dig in and probably gain a little understanding.

```
(py3) ~/repo/cockroachdb/university-multi-region-course-app-exercises ga_updates % grep -Iir '26257' .
./solutions/08-placement-restricted/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=12 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=d:region=us-central,zone=f:region=eu-west,zone=a:region=eu-west,zone=d:region=eu-west,zone=f --empty
./solutions/02-poor-resilience/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=9 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=b:region=us-central,zone=c --empty
./solutions/07-regional-tables/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=9 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=d:region=us-central,zone=f --empty
./solutions/00-initial-state/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database
./solutions/09-tradeoffs/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=12 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=d:region=us-central,zone=f:region=eu-west,zone=a:region=eu-west,zone=d:region=eu-west,zone=f --empty
./solutions/05-global-tables/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=9 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=d:region=us-central,zone=f --empty
./solutions/03-implement-multi-region-db/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=9 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=d:region=us-central,zone=f --empty
./solutions/06-regional-by-row/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=9 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=d:region=us-central,zone=f --empty
./solutions/01-identifying-necessary-regions/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=9 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=sa-east,zone=b:region=eu-central,zone=c --empty
./solutions/04-survival-goals/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database --nodes=9 --demo-locality=region=us-east,zone=a:region=us-east,zone=b:region=us-east,zone=c:region=us-west,zone=b:region=us-west,zone=c:region=us-west,zone=d:region=us-central,zone=a:region=us-central,zone=d:region=us-central,zone=f --empty
./exercises/movr/rides/src/main/resources/application.yaml:    url: jdbc:postgresql://localhost:26257/movr_rides
./exercises/movr/users/src/main/resources/application.yaml:    url: jdbc:postgresql://localhost:26257/movr_users
./exercises/movr/cockroach/run.sh:cockroach demo --sql-port 26257 --no-example-database
(py3) ~/repo/cockroachdb/university-multi-region-course-app-exercises ga_updates %
```

Reply:
This is a dirty little trick to show the `vehicles` service unable to connect after killing nodes in a specific region. Since we're using `demo`, it's simulated with services directly connecting to the nodes rather than a load balancing as in regular life. I can't have vehicles running on `26257` because if you kill node 1 (which I was doing before) you basically kill your connection to `demo` in the terminal because `demo` connects to `26257` by default. So I connect `vehicles` to `26263` instead so that you can continue to interact with `demo` after the node running `vehicles` is killed.

Reply:
Clever!